### ISSUE TITLE
Did Uniswap abandon this repo?

### ISSUE THREAD

**Issue by floticerus** [NoahZinsmeister/web3-react#927]
It sure looks that way.

Are there any forks that are maintained?

**Comment by eastedgeZack** [comment]
It seems that way. Having problems with metadata errors for walletconnect

**Comment by goffxnca** [comment]
yeah, it looks like it is. some of the issues have been pending since 2021. that’s quite big a while.
